config/environments/development.rb

```diff
@@ -41,6 +41,23 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :sendmail
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_options = { from: 'example@email.com' }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
+    domain: 'example.com',
+    user_name: Rails.application.credentials.gmail[:email],
```

Comment:
- [ ] Kindly fix this error 
![image](https://user-images.githubusercontent.com/62705947/201356829-6ed21ce3-8051-4668-8a86-4aa633ccab60.png)